packages/expansion-hub/src/start-keep-alive.ts

```diff
@@ -0,0 +1,18 @@
+import {ExpansionHubInternal} from "./internal/ExpansionHub";
+
+/**
+ * Starts a keep alive task for a given hub.
+ * @param hub hub to start the keep alive for
+ * @param interval interval in ms between each keep alive
+ */
+export function startKeepAlive(hub: ExpansionHubInternal, interval: number) {
```

Comment:
Let's rename `interval` to `intervalMs`.

I don't think it makes sense to export this method. However, it would be reasonable for software to want to control exactly when (or even if*) keep alives are sent, so at some point it might be nice to make keep alive sending be optional or to specify the frequency, and I think we should expose the `sendKeepAlive()` method (which I believe we already do).

*If an application knows that it's constantly sending other messages, KeepAlives aren't needed.